doc-content/enterprise-only/employee-rostering/optashift-ER-openshift-deploy-con.adoc

```diff
@@ -0,0 +1,19 @@
+[id='optashift-ER-openshift-deploy-proc']
```

Comment:
This looks more link a concept module than a procedure. Change the numbered bullets to unnumbered (*). These are not steps to be performed sequentially.